### PR TITLE
Bookify basic discrete math

### PR DESCRIPTION
**Pull request by gyosh**
[matdis.pdf](https://github.com/ia-toki/training-gate-id-book/files/1443803/matdis.pdf)

### CODE REVIEW

File: topik/pemrograman-kompetitif-dasar/08-perkenalan-graph/pemrograman-kompetitif-dasar-08-perkenalan-graph-book.tex [hunk 1]
```diff
@@ -354,6 +354,11 @@ \section{Keuntungan dan Kerugian Representasi Graf (lanj.)}
   \item Jika ada edge dari A ke B yang akan dibuang, maka pada adjacency matrix kita hanya perlu mengubah nilai Matrix[A][B]. Sedangkan pada adjacency list, harus dilakukan iterasi terlebih dahulu untuk mencari edge tersebut, lalu menghapusnya juga lebih rumit karena harus menggeser seluruh elemen setelahnya.
   \item Untuk mencari tetangga-tetangganya, maka pada adjacency matrix perlu dilakukan iterasi terhadap seluruh node yagn ada. Sementara itu, pada adjacency list hanya perlu dilakukan iterasi pada list yang isinya merupakan tetangga dari node yang bersangkutan.
 \end{itemize}
+<<<<<<< Updated upstream
```
Comment (fushar) [inline topik/pemrograman-kompetitif-dasar/08-perkenalan-graph/pemrograman-kompetitif-dasar-08-perkenalan-graph-book.tex]: Goz goz ini apa goz

File: topik/pemrograman-kompetitif-dasar/02-matematika-diskret-dasar/pemrograman-kompetitif-dasar-02-matematika-diskret-dasar-book.tex [hunk 2]
```diff
@@ -1,53 +1,26 @@
...
-\section{Matematika Diskret}
-\begin{itemize}
-  \item Merupakan cabang matematika yang mempelajari tentang sifat bilangan bulat, logika, kombinatorika, dan graf.
-\end{itemize}
+Matematika diskret merupakan cabang matematika yang mempelajari tentang sifat bilangan bulat, logika, kombinatorika, dan graf. Ilmu pada jenis matematika ini digunakan sebagai dasar ilmu komputer secara umum. Pada bagian ini, kita akan mencakup dasar ilmu matematika diskret yang umum digunakan.
 
 \section{Arimetika Modular}
 
-\section{Konsep Modulo}
+Modulo merupakan suatu operator matematika untuk mendapatkan sisa bagi suatu bilangan terhadap suatu bilangan lainnya. Operasi modulo dilambangkan dengan $\bmod$. Operasi $a \bmod m$ biasa dibaca "$a$ modulo $m$", dan memberikan sisa hasil bagi $a$ oleh $m$. Sebagai contoh:
```
Comment (fushar) [inline topik/pemrograman-kompetitif-dasar/02-matematika-diskret-dasar/pemrograman-kompetitif-dasar-02-matematika-diskret-dasar-book.tex]: "Modulo" di awal di-bold gak sih, soalnya term baru

File: topik/pemrograman-kompetitif-dasar/02-matematika-diskret-dasar/pemrograman-kompetitif-dasar-02-matematika-diskret-dasar-book.tex [hunk 3]
```diff
@@ -60,87 +33,64 @@ \section{Aplikasi Modulo (lanj.)}
...
 
-\section{Uji Keprimaan (lanj.)}
-\begin{itemize}
-  \item Ada solusi yang lebih cepat dari $O(N)$.
-  \item Manfaatkan observasi bahwa jika $N = a \times b$, dan $a \leq b$, maka $a \leq \sqrt{N}$ dan $b \geq \sqrt{N}$.
-  \item Kita tidak perlu memeriksa $b$; seandainya $N$ habis dibagi $b$, tentu $N$ habis dibagi $a$.
-  \item Jadi kita hanya perlu memeriksa hingga $\sqrt{N}$.
-  \item Kompleksitas: $O(\sqrt{N})$
-\end{itemize}
+Terdapat solusi uji keprimaan yang lebih cepat dari $O(N)$. Manfaatkan observasi bahwa jika $N = a \times b$, dan $a \leq b$, maka $a \leq \sqrt{N}$ dan $b \geq \sqrt{N}$. Kita tidak perlu memeriksa $b$; seandainya $N$ habis dibagi $b$, tentu $N$ habis dibagi $a$. Jadi kita hanya perlu memeriksa hingga $\sqrt{N}$. Jadi kompleksitas waktunya adalah $O(\sqrt{N})$.
```
Comment (fushar) [inline topik/pemrograman-kompetitif-dasar/02-matematika-diskret-dasar/pemrograman-kompetitif-dasar-02-matematika-diskret-dasar-book.tex]: daripada

File: topik/pemrograman-kompetitif-dasar/02-matematika-diskret-dasar/pemrograman-kompetitif-dasar-02-matematika-diskret-dasar-book.tex [hunk 4]
```diff
@@ -150,24 +100,13 @@ \section{Uji Keprimaan (lanj.)}
...
-  \item Solusi:
-  \begin{enumerate}
-    \item Lakukan iterasi dari 2 sampai N
-    \item Untuk tiap bilangan, cek apakah dia bilangan prima. Jika ya, kita bisa memasukkannya ke daftar bilangan prima.
-  \end{enumerate}
-\end{itemize}
+\section{Prime Generation (Pembangkitan Bilangan Prima)}
 
-\section{Solusi Naif (lanj.)}
+Aktivitas untuk mendapatkan bilangan-bilangan prima disebut dengan pembangkitan bilangan prima. Misalkan kita ingin mengetahui himpunan bilangan prima yang tidak lebih dari $N$. Solusi sederhananya adalah membangkitkan bilangan prima dengan iterasi dan uji keprimaan:
```
Comment (fushar) [inline topik/pemrograman-kompetitif-dasar/02-matematika-diskret-dasar/pemrograman-kompetitif-dasar-02-matematika-diskret-dasar-book.tex]: daripada

**Comment by gyosh** [comment]
PDF terbaru: [out-193-joined.pdf](https://github.com/ia-toki/training-gate-id-book/files/1443822/out-193-joined.pdf)

**Pull request merged**